game-core/src/main/java/games/strategy/triplea/pbem/AxisAndAlliesForumPoster.java

```diff
@@ -1,241 +1,180 @@
-            status = response2.getStatusLine().getStatusCode();
-            body = EntityUtils.toString(response2.getEntity());
-            if (status == HttpURLConnection.HTTP_MOVED_TEMP) {
-              // site responds with a 302 redirect back to the forum index (board=40)
-              // The syntax for post is ".....topic=xx.yy" where xx is the thread id, and yy is the post number in the
-              // given thread
-              // since the site is lenient we can just give a high post_number to go to the last post in the thread
-              turnSummaryRef = UrlConstants.AXIS_AND_ALLIES_FORUM + "?topic=" + m_topicId + ".10000";
-            } else {
-              // these two patterns find general errors, where the first pattern checks if the error text appears,
-              // the second pattern extracts the error message. This could be the "The last posting from your IP was
-              // less
-              // than 15 seconds
-              // ago.Please try again later"
-              // this patter finds errors that are marked in red (for instance "You are not allowed to post URLs", or
-              // "Some one else has posted while you vere reading"
-              Matcher matcher = ERROR_LIST_PATTERN.matcher(body);
-              if (matcher.matches()) {
-                throw new Exception("The site gave an error: '" + matcher.group(1) + "'");
-              }
-              matcher = AN_ERROR_OCCURRED_PATTERN.matcher(body);
-              if (matcher.matches()) {
-                matcher = ERROR_TEXT_PATTERN.matcher(body);
-                if (matcher.matches()) {
-                  throw new Exception("The site gave an error: '" + matcher.group(1) + "'");
-                }
-              }
-              final Header refreshHeader = response2.getFirstHeader("Refresh");
-              if (refreshHeader != null) {
-                // sometimes the message will be flagged as spam, and a refresh url is given
-                // refresh: 0; URL=http://...topic=26114.new%3bspam=true#new
-                final String value = refreshHeader.getValue();
-                final Pattern p =
-                    Pattern.compile("[^;]*;\\s*url=.*spam=true.*", Pattern.DOTALL | Pattern.CASE_INSENSITIVE);
-                m = p.matcher(value);
-                if (m.matches()) {
-                  throw new Exception("The summary was posted but was flagged as spam");
-                }
-              }
-              throw new Exception(
-                  "Unknown error, please contact the forum owner and also post a bug to the tripleA development team");
-            }
-          }
-        } else {
-          throw new Exception("Unable to load forum post " + m_topicId);
+  private static void deleteToken(final CloseableHttpClient client, final int userId, final String token)
+      throws IOException {
+    final HttpDelete httpDelete = new HttpDelete(forumUrl + "/api/v2/users/" + userId + "/tokens/" + token);
```

Comment:
There's a missing `HttpProxy.addProxy(httpDelete)` call below this line that exists in `NodeBbForumPoster` on `master`.  However, as this bug currently exists in 1.9, has existed for quite some time, and no one has complained about it (probably because we don't have many PBF users behind a proxy), not sure it's important to follow-up with another PR on this branch.

Reply:
Should be fine without it.